toolkits/landcover/impl/bands.js

```diff
@@ -17,7 +17,7 @@
  * @fileoverview Static functions for band transformations.
  */
 
- var NamedArgs = require('users/google/toolkits:landcover/impl/named-args').NamedArgs;
+ var NamedArgs = require('users/google/toolkits:landcover/impl/named-args.js').NamedArgs;
```

Comment:
extra leading whitespace here.

Reply:
Are you referring to this?
![image](https://user-images.githubusercontent.com/228050/66585227-7ab56800-eb54-11e9-85b5-dcbc029552bf.png)

If so it's actually part of GitHub's diff gutter (this has tricked me in the past as well).

Reply:
Yes, I'm referring to this line, but it's not github.  There's an extra leading space on this line.
Compare this to the next one:
![image](https://user-images.githubusercontent.com/53860502/66587460-b8cd7e80-eb8a-11e9-9e54-6e57bc1a846e.png)

![image](https://user-images.githubusercontent.com/53860502/66587483-c551d700-eb8a-11e9-9213-6285bc751e95.png)

Reply:
Wow, good eye. Thanks!